assets/panel/debugger.properties

```diff
@@ -240,6 +240,10 @@ breakpointMenuItem.disableAll=Disable all breakpoints
 breakpointMenuItem.disableAll.accesskey=k
 breakpointMenuItem.deleteAll=Remove all breakpoints
 breakpointMenuItem.deleteAll.accesskey=a
+breakpointMenuItem.removeCondition=Remove breakpoint condition
+breakpointMenuItem.removeCondition.accesskey=c
+breakpointMenuItem.editCondition=Edit breakpoint condition
```

Comment:
breakpointMenuItem.editCondition.label

Reply:
so like this?
breakpointMenuItem.removeCondition.label=Remove breakpoint condition
breakpointMenuItem.editCondition.label=Edit breakpoint condition
(and delete acceskey ones)

Reply:
No, like this

```
breakpointMenuItem.removeCondition.label=Remove breakpoint condition
breakpointMenuItem.removeCondition.accesskey=c
+breakpointMenuItem.editCondition.label=Edit breakpoint condition
+breakpointMenuItem.editCondition.accesskey=n
```